### DISCUSSION TITLE
Unable to find modules after installation.

### DISCUSSION THREAD

**Discussion by hayley-wragg** [firedrakeproject/firedrake#2602] [Firedrake support] [answered]
I have run the installation using:
curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/master/scripts/firedrake-install
python3 firedrake-install.

[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/9863428/firedrake-install.log)
completed without a problem.

When I then run the tests:
source firedrake/bin/activate
cd src/firedrake
pytest tests/regression/ -k "poisson_strong or stokes_mini or dg_advection"

I get "no module pyadjoint".

When I tried a basic firedake example:

>from firedrake import *
>RectangleMesh(10,10,2,2)

I get "RectangleMesh is not defined"

I tried removing the installation and rerunning but the same happened again. I can't find any clues as to why this is happening and how I can vary the installation to prevent it.

**Comment by ksagiyam** [comment, marked as answer]
The install log indicates that your install was not successful.

I can see warnings like:
```
Warning: Type mismatch between actual argument at (1) and actual argument at (2) (REAL(8)/INTEGER(4)).
```
Those are "Warning"s (not "Error"s), but could you try setting the following environment variables before installing:
```
export FCFLAGS="-w -fallow-argument-mismatch"
export FFLAGS="-w -fallow-argument-mismatch"
```
?

See also: https://github.com/Unidata/netcdf-fortran/issues/212

**Reply by hayley-wragg** [reply]
Thank you. That seems to have fixed that problem! Now onto the next one..
